app/components/canary-test-result-summary.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+
+function computedDivide(a, b) {
```

Comment:
I think this could have a different name -- computedDivide makes me think it would return a number, not a formatted string.